templates/injector-service.yaml

```diff
@@ -11,7 +11,8 @@ metadata:
   {{ template "injector.service.annotations" . }}
 spec:
   ports:
-  - port: 443
+  - name: {{ include "vault.scheme" . }}
```

Comment:
The injector is always HTTPs and isn't based on `vault.scheme`. So this can be hard coded to `https`.

Reply:
Done